conpot/tests/test_mac_addr.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2014  Lukas Rist <glaslos@gmail.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import unittest
+import random
+import conpot.utils.mac_addr
+import subprocess
+import re
+
+
+class TestMacAddrUtil(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_mac(self):
+        for i in range(5):
+                randmac = ":".join(
+                    ["%02x" % (random.randint(1, 255)) for i in range(6)])
+                s = subprocess.Popen(["spoof-mac.py", "list"], stdout=subprocess.PIPE)
+                iface = re.findall(r'"([A-Za-z0-9]*)"', s.stdout.readlines()[0])[1]
```

Comment:
What if there is no such device if we hardcode the interface name ? The test will fail right ? You mean eth[0-9]* ??